tasks/package.js

```diff
@@ -29,23 +80,40 @@ var fileExists = function(filepath) {
 };
 
 module.exports = function() {
-  var output = fs.createWriteStream(
-    'package-' + extensionDescriptor.name + '-' + extensionDescriptor.version + '.zip'
-  );
-  var zipArchive = archiver('zip');
-
-  zipArchive.pipe(output);
-
   var filepaths = getPaths(extensionDescriptor);
+  var outputLocation = getOutputLocation();
 
-  filepaths.forEach(function(filepath) {
-    if (!fileExists(filepath)) {
-      var error = 'Cannot find file: ' + filepath;
-      console.error(chalk.red(error));
-      process.exit(1);
+  if (argv.dryRun) {
```

Comment:
For testing purposes of the `--out` command line option, it was convenient to "dry run" where things would get written without actually making a zip. Therefore, I added it as something we could do all of the time.